src/GitExtensions.GerritPlugin/GitExtensions.GerritPlugin.csproj

```diff
@@ -5,15 +5,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="System.ComponentModel.Composition" />
-    <Reference Include="System.Windows.Forms" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="JetBrains.Annotations" Version="2021.2.0">
+    <PackageReference Include="JetBrains.Annotations" Version="2021.3.0">
```

Comment:
You may want to consider use Roslyn annotations instead of those provided by JetBrains.